src/main/python/server.py

```diff
@@ -37,11 +67,24 @@ def serve(port):
         server.stop(0)
 
 
+def _get_server(port):
+    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+    service_pb2_grpc.add_FeatureExtractorServicer_to_server(Service(), server)
+    server.add_insecure_port('[::]:%d' % port)
+    return server
+
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Feature Extractor Service.')
     parser.add_argument("--port", type=int, default=9001,
                         help="server listen port")
     args = parser.parse_args()
 
+    # sourced-ml expects PYTHONHASHSEED != random or unset
+    if os.getenv("PYTHONHASHSEED", "random") == "random":
```

Comment:
As far as I can see, these are src-d/ml implementation details, relevant to some extractors. 
What do you think - on our side, would it be wise to make them part of the API and pass them as explicit parameters? 

This could help to decouple a library API with it's runtime environment, which is a good practice anyway.

Reply:
Nope. This parameter changes how python interpreter works. You can’t change it in runtime. That’s why the check is in __main__.

Reply:
Reading https://docs.python.org/3.3/using/cmdline.html#envvar-PYTHONHASHSEED helped to understand it better, which would be nice to have somewhere (i.e in that comment above the code)

Reply:
there is a comment already. 2 lines below.